tests/unit/network/test__init__.py

```diff
@@ -0,0 +1,65 @@
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Test __init__.py for network interfaces to Cylc scheduler objects."""
+
+import pytest
+from _pytest.monkeypatch import MonkeyPatch
+
+import cylc
+from cylc.flow.exceptions import CylcVersionError
+from cylc.flow.network import get_location
+from cylc.flow.workflow_files import load_contact_file, ContactFileFields
+
+
+BASE_CONTACT_DATA = {
+    ContactFileFields.HOST: 'foo',
+    ContactFileFields.PORT: '42',
+}
+
+
+@pytest.fixture(scope='module')
```

Comment:
This can be a regular function scoped fixture to save the _pytest business.

(module scoping is useful when fixtures take a while to setup).

Reply:
Fair point.